dynamic_analysis_pkg/dynamic_analysis_pkg/__init__.py

```diff
@@ -0,0 +1,167 @@
+
+from os import makedirs
+from os.path import join, exists, isdir, basename
```

Comment:
You should take the habit of using `os.path` big brother built in in python 3:
```suggestion
from pathlib import Path
```
Then wrap all you path into `Path` obects:
```python
my_path = Path(my_path_which_was_a_string)

# replace 
# exists by:
my_path.exists()

# mkdirs by:
my_path.mkdir() # there is a flag if you want/don't want to raise when it already exists

# join by:
my_sub_dir_path = my_path / 'sub-dir'

# is_dir by:
my_path.is_dir()

# basename by:
my_path.name
```

This is probably the best addition to python3, read more at:
https://docs.python.org/3/library/pathlib.html